appveyor.yml

```diff
@@ -0,0 +1,32 @@
+# version format
+version: 1.0.{build}-{branch}
+
+# Do not build on tags (GitHub and BitBucket)
+skip_tags: true
+
+# Do not build feature branch with open Pull Requests
+skip_branch_with_pr: true
```

Comment:
This settings prevents the double build of a branch with an open pull request. With Travis CI the PR is built and the branch is built. With AppVeyor the default behavior is similar unless this setting is used.